irksome/manipulation.py

```diff
@@ -0,0 +1,194 @@
+"""Manipulation of expressions containing :class:`~.TimeDerivative`
```

Comment:
Do we want to put in some kind of tests for this now, or save that until we start doing Galerkin-type manipulations down the road?

Reply:
Added some tests.